spec.bs

```diff
@@ -130,110 +129,85 @@ are:
     is not in the range [0, 2<sup>128</sup>−1].
 1. Throw {{RangeError}} if |contribution|["{{PAHistogramContribution/value}}"]
     is negative.
-1. [=list/Append=] |contribution| to the [=contributions cache=].
-
-Issue: Check that value can actually be zero in the spec pipeline.
-
-
-Exposing to the Shared Storage API {#shared-storage}
-====================================================
-
-<xmp class="idl">
-partial interface SharedStorageWorkletGlobalScope {
-  readonly attribute PrivateAggregation privateAggregation;
-};
-</xmp>
-
-Immediately after an operation completes, [=process the Shared Storage
-contributions cache=] given the worklet's [=contributions cache=] and the
-worklet's global scope.
-
-Issue: How to handle batching different operation invocations properly. This
-    doesn't work for simultaneous operations. Also disallow usage outside an
-    operation.
-
-To <dfn>process the Shared Storage contributions cache</dfn> given a [=list=]
-|contributionsCache| and a {{SharedStorageWorkletGlobalScope}} |scope|, perform
-the [=PrivateAggregation/report creation and scheduling steps=] with |scope|'s
-[=relevant settings object=]'s [=environment settings object/origin=],
-"<code>[=context type/shared-storage=]</code>" and |contributionsCache|.
-
-
-Exposing to the Protected Audience API {#protected-audience}
-============================================================
-
-<xmp class="idl">
-partial interface InterestGroupScriptRunnerGlobalScope {
-  readonly attribute PrivateAggregation privateAggregation;
-};
+1. Let |batchingScopeSteps| be the {{PrivateAggregation}} object's
+    [=PrivateAggregation/get batching scope steps=].
+1. If |batchingScopeSteps| is null, throw a new {{DOMException}} with name
+    "`NotAllowedError`".
+    Note: This indicates the API is not yet available, for example, because the
```

Comment:
Style nit: try adding an extra empty line between the note and the algorithm step. As is, it's kinda mashed together..